consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/payload/PreparePayload.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.consensus.qbft.payload;
+
+import org.hyperledger.besu.consensus.common.bft.ConsensusRoundIdentifier;
+import org.hyperledger.besu.consensus.common.bft.payload.Payload;
+import org.hyperledger.besu.consensus.qbft.messagedata.QbftV1;
+import org.hyperledger.besu.ethereum.core.Hash;
+import org.hyperledger.besu.ethereum.rlp.RLPInput;
+import org.hyperledger.besu.ethereum.rlp.RLPOutput;
+
+import java.util.Objects;
+import java.util.StringJoiner;
+
+public class PreparePayload implements Payload {
+  private static final int TYPE = QbftV1.PREPARE;
+  private final ConsensusRoundIdentifier roundIdentifier;
+  private final Hash digest;
+
+  public PreparePayload(final ConsensusRoundIdentifier roundIdentifier, final Hash digest) {
+    this.roundIdentifier = roundIdentifier;
+    this.digest = digest;
+  }
```

Comment:
Leaving sender out for now? We can always add it in easily I guess it becomes a required field

Reply:
I'd like to leave it out - i.e. it shouldn't need to go "on the line" - it can be derived from the signature.